README.md

```diff
@@ -195,6 +195,6 @@ Specifies whether to collect statistic with respect to timing, size of constrain
 For example, generic solver can be invoked through following command:
 
 ````
-./scripts/inference --mode INFER --checker ostrusted.OsTrustedChecker --solver checkers.inference.solver.GeneralSolver --solverArgs solver=MaxSAT,useGraph=true,collectStatistics=true,solveInParallel=false [List of files]
+./scripts/inference --mode INFER --checker ostrusted.OsTrustedChecker --solver checkers.inference.solver.GeneralSolver --solverArgs solver=MaxSat,useGraph=true,collectStatistics=true,solveInParallel=false [List of files]
```

Comment:
I don't see a 'GeneralSolver' class. Does this command actually work after this change?
Can you make a new PR that adds a simple test for this command?

Reply:
No, the command in the example doesn't work, for it throws `ClassNotFoundException` when trying to initialize a `GeneralSolver`. 

Usually I use `--solver checkers.inference.solver.SolverEngine` and specify the backend solver as `MaxSat` in the `--solverArgs`. I think this could be the typical use case to be put into the instruction document.

Reply:
Shall we remove `GeneralSolver` from README?